packages/grafana-ui/src/components/uPlot/Plot.tsx

```diff
@@ -123,7 +82,7 @@ export const UPlotChart: React.FC<PlotProps> = props => {
 
   // Memoize plot context
   const plotCtx = useMemo(() => {
-    return buildPlotContext(registerPlugin, canvasRef, props.data, plotInstance);
+    return buildPlotContext(registerPlugin, addSeries, addAxis, addScale, canvasRef, props.data, plotInstance);
   }, [registerPlugin, canvasRef, props.data, plotInstance]);
```

Comment:
nit: same comment as above but should `addSeries, addAxis, addScale` be added to dependencies?

Reply:
yes, great catch @hugohaggmark!